### ISSUE TITLE
error with webpack-hot-client

### ISSUE THREAD

**Issue by jeetiss** [c8r/kit#169]
Hi ^_^

when kit-cli installed globally, command `kit example` runs with error

```bash
ERROR in multi webpack-hot-client/client?8e5c088f-87d7-482f-8ad1-48719db0177c /Users/jeetiss/n/lib/node_modules/@compositor/kit-cli/node_modules/@compositor/x0/lib/entry
Module not found: Error: Can't resolve 'webpack-hot-client/client?8e5c088f-87d7-482f-8ad1-48719db0177c' in '/Users/jeetiss/Projects/systemo/stories'
 @ multi webpack-hot-client/client?8e5c088f-87d7-482f-8ad1-48719db0177c /Users/jeetiss/n/lib/node_modules/@compositor/kit-cli/node_modules/@compositor/x0/lib/entry
```

and if install `kit-cli` in local ./node_modules works fine

i can find issue with webpack-hot-client, and this should fix problem
webpack-contrib/webpack-hot-client#8

**Comment by ferdinandsalis** [comment]
I am getting the same error …

**Comment by johno** [comment]
Thanks for the report! Do you mind providing your `kit`, `npm`, and `node` versions please?

Have you also tried running the cli via your `package.json`? I'm wondering if this error is specific to global installs.

**Comment by ferdinandsalis** [comment]
@johno indeed this only occurs with global installs. It works when it is a local dependency and that is how I am using it now. These are the versions I am using `node 9.11.1`, `npm 6.1.0` and `kit 1.0.37`.

Anyway thanks for kit. I love the simplicity of it.

**Comment by beeant** [comment]
I'm having the same problem...

**Comment by jxnblk** [comment]
Thanks for reporting this! I can confirm that I have seen the same issue on one computer, but not the other. In #179 I'm attempting to change the webpack resolve as mentioned in webpack-contrib/webpack-hot-client#62, and have published a prerelease. If anyone can help by trying to install the following to confirm that it works as a global install, that'd be super helpful

```sh
npm i -g @compositor/kit-cli@alpha
```

**Comment by jeetiss** [comment]
I have some errors with resolving `babel-runtime`

```
ERROR in ../components/Button.js
Module not found: Error: Can't resolve 'babel-runtime/helpers/taggedTemplateLiteral' in '/Users/jeetiss/Projects/systemo/components'
 @ ../components/Button.js 7:30-84
 @ ./Button.js
 @ . sync nonrecursive \.(js|mdx|jsx)$
 @ /Users/jeetiss/n/lib/node_modules/@compositor/kit-cli/node_modules/@compositor/x0/lib/entry.js
 @ multi /Users/jeetiss/n/lib/node_modules/@compositor/kit-cli/node_modules/@compositor/x0/lib/entry
```

**Comment by jeetiss** [comment]
I fix this in #180

**Comment by johno** [comment]
We just published another release that should _hopefully_ solve the `webpack-hot-client` error once and for all.

I believe the issue is similar to the `babel-runtime` resolution as well. The underlying issue appears to be a result of `@compositor/x0` sometimes being nested within `@compositor/kit-cli/node_modules` and sometimes not.

Haven't a clue why this is occurring, but was able to fix it for `webpack-hot-client` via c8r/x0#67 since it added an alias that relied on a flat installation (which isn't always the case).

**Comment by johno** [comment]
Going to close this as it seems to be fixed. If anyone encounters this error with the latest version of kit-cli please open up a new issue.
